src/domain/frontend/log.ts

```diff
@@ -1,4 +1,5 @@
 /** ログ出力用の関数を生成する。ブラウザ専用。 */
+// TODO: DIできるようにする
```

Comment:
目的語があるとコメントが不要になったかどうかの判断しやすいかも
```suggestion
// TODO: window.backendをDIできるようにする
```